### DISCUSSION TITLE
Documentation for adding a new tool?

### DISCUSSION THREAD

**Discussion by poperigby** [mason-org/mason.nvim#186] [Q&A] [answered]
I'm trying to add a new language server, but I don't see any documentation on how to do this. I've looked at some of the other installation scripts, but I'm a bit lost.

**Comment by williamboman** [comment, marked as answer]
Hello! Have you seen the following: 

- https://github.com/williamboman/mason.nvim/blob/main/CONTRIBUTING.md#package-installer
- https://github.com/williamboman/mason.nvim/blob/main/doc/reference.md

**Reply by poperigby** [reply]
There we go. Thank you!
